libs/core/src/components.d.ts

```diff
@@ -19,6 +19,10 @@ export namespace Components {
           * @defaultValue false
          */
         "badge"?: boolean;
+        /**
+          * String used for the component `id` attribute.
```

Comment:
The word "string" used here is redundant as the type is set to be "string".  I personally like the previous description that was used for `input`.

How is this, it's a meet-in-the-middle kind of approach. 
> A unique identifier used for the underlying component `id` attribute.

Reply:
Done.